tests/test_text_extraction.py

```diff
@@ -189,3 +189,20 @@ def test_layout_mode_warnings(mock_logger_warning):
     mock_logger_warning.assert_called_with(
         "Argument visitor_text is ignored in layout mode", "pypdf._page"
     )
+
+
+@pytest.mark.enable_socket()
+def test_space_with_one_unit_smaller_than_font_width():
+    """Tests for #1328"""
+    url = "https://github.com/py-pdf/PyPDF2/files/9498481/0004.pdf"
+    name = "iss1328.pdf"
+    reader = PdfReader(BytesIO(get_data_from_url(url, name=name)))
+    page = reader.pages[0]
+    extracted = page.extract_text().split("Description:")[1].split("8/11/22")[0].strip()
```

Comment:
Why splitting instead of using plain `assert expected in extracted`?

Reply:
Due to the length of the test data presented, the comparison was made to the issue (#1328) indicated area.
If there is a problem with the methodology, I will consider modifying it.

Reply:
I do not generally consider this a problem. This might lead to strange/non-obvious test failures, while otherwise we get the complete output, but possibly lots of output.

Reply:
Thank you.
Does this mean that it is better to check the full text?
Sorry, I'm not good at English.

Reply:
I understand. So why not simply use in? I'll fix it.

Reply:
Please confirm that it has been changed.